spring-boot/admin/src/main/resources/templates/admin/fragments/gateway_health_monitor.html

```diff
@@ -0,0 +1,85 @@
+                                    <span th:text="#{admin.dashboard.gatewayhealthmonitor.state.ok}"></span>
+                                    <span class="everything-ok-symbol">&#10004;</span></span>
+                    <span th:case="${T(eu.coatrack.api.ProxyHealthStatus).NEVER_CONNECTED}">
+                                    <span th:text="#{admin.dashboard.gatewayhealthmonitor.state.notconnected}"></span>
+                                    <span class="warning-not-connected-exclamation-mark">&#9888;</span>
+                                    </span>
+                </div>
+
+                <table class="health-monitor-data-table">
+                    <tr>
+                        <th th:text="#{admin.dashboard.gatewayhealthmonitor.gatewayname}"></th>
+                        <th th:text="#{admin.dashboard.gatewayhealthmonitor.status}"></th>
+                        <th th:text="#{admin.dashboard.gatewayhealthmonitor.lastcontact}"></th>
+                        <th th:text="#{admin.dashboard.gatewayhealthmonitor.notification.switch}"></th>
+                    </tr>
+                    <tr th:each="gateway : ${gatewayHealthMonitorProxyData.gatewayDataListForGatewayHealthMonitor}">
+                        <td align="left" th:text="${gateway.name}"></td>
+
+                        <td align="center" th:switch="${gateway.status}">
+                            <span th:case="${T(eu.coatrack.api.ProxyHealthStatus).WARNING}">
+                                <span th:class="'neutral-state-symbol warning-state'"></span></span>
+                            <span th:case="${T(eu.coatrack.api.ProxyHealthStatus).CRITICAL}">
+                                <span th:class="'neutral-state-symbol critical-state'"></span></span>
+                            <span th:case="${T(eu.coatrack.api.ProxyHealthStatus).OK}">
+                                <span th:class="'neutral-state-symbol ok-state'"></span></span>
+                            <span th:case="${T(eu.coatrack.api.ProxyHealthStatus).NEVER_CONNECTED}">
+                                <span th:class="'neutral-state-symbol'"></span></span>
+                            <span th:case="${T(eu.coatrack.api.ProxyHealthStatus).IGNORE}">
+                                <span th:class="'neutral-state-symbol ignore-state'"></span></span>
+                        </td>
+
+                        <th:block th:if="${gateway.isMonitoringEnabled}">
+                            <td align="left" th:if="${gateway.minutesPassedSinceLastContact != null}"
+                                th:text="${gateway.minutesPassedSinceLastContact &lt; 1 ? 'Less than 1 minute ago' : gateway.minutesPassedSinceLastContact + ' min ago'}">
+                            </td>
+                            <td align="left" th:if="${gateway.minutesPassedSinceLastContact == null}"
+                                th:text="#{admin.dashboard.gatewayhealthmonitor.notconnectedgateway.text}"></td>
```

Comment:
Please move closing tag to the next line to improve readability.

Reply:
Done